<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AllTests</class>
 <widget class="QDialog" name="AllTests">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1018</width>
    <height>653</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MODEL TEST</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>145</y>
     <width>761</width>
     <height>311</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetNoConstraint</enum>
    </property>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <item row="3" column="0">
       <widget class="QCheckBox" name="checkBox">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>DejaVu Sans</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QCheckBox" name="checkBox_3">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>DejaVu Sans</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QCheckBox" name="checkBox_2">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QCheckBox" name="checkBox_4">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>DejaVu Sans</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QTextEdit" name="textEdit">
        <property name="font">
         <font>
          <family>MV Boli</family>
          <pointsize>18</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>500</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Next </string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>460</y>
     <width>211</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>FINISH TEST</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>20</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS UI Gothic</family>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Choose one option.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>20</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Time Elapsed:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>460</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Press Next to Start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>470</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>CORRECT ANSWER</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>510</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>570</y>
     <width>1011</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_pic">
   <property name="geometry">
    <rect>
     <x>-223</x>
     <y>-75</y>
     <width>1351</width>
     <height>761</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_pic</zorder>
  <zorder>verticalLayoutWidget</zorder>
  <zorder>pushButton</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>lineEdit</zorder>
  <zorder>lineEdit_2</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>pushButton_3</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
